### PR TITLE
Show a tick in success messages #7058

### LINKED ISSUE #7058: Show a tick in success messages
image

Similar to how we show an `i` icon in the info message in screenshot, we can also show a tick icon in success message.

image

### PR DESCRIPTION
**Pull request by Mynk96**
Fixes #7058 

**Outline of Solution**
Added ok-sign glyphicon for success messages in statusMessage.tag

### CODE REVIEW

**Comment by samsontmr** [comment]
Could you add a screenshot showing what your change looks like?

**Comment by Mynk96** [comment]
gly

screenshot

**Comment by Mynk96** [comment]
gly1

screenshot

**Comment by samsontmr** [comment]
You will need to resolve build failures before this PR can be merged.

**Comment by Mynk96** [comment]
@samsontmr ready for review

**Comment by damithc** [comment]
Good work so far @Mynk96 
Some non-expert questions,
1. What if we add similar icons to other types of status messages at the same time? It might give a more optimized code structure rather than add one at a time.
2. What is the possibility of adding icons at css level, instead of adding it in the jstl code. Is that even possible?

**Comment by Mynk96** [comment]
@damithc 
1) I completely agree with you infact I was thinking of suggesting you for the same.
2)Yes it is possible to add icons at css level.
Adding
.alert-success:before {
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding-right:2px;
  font-family: 'Glyphicons Halflings'; 
  content:"\e084";
}
in teammatescommon.css produces the same change required.Also in my opinion css approach will be better as it keeps the JSTL code clean and using this approach we can easily add icons for all the alerts.

**Comment by damithc** [comment]
@wkurniawan07 what do you think? Go for the css approach?

**Comment by whipermr5** [comment]
I much prefer the CSS approach haha. In addition to being cleaner, using the CSS pseudo-element (`:before`) approach will [hide the icon from screen readers](http://www.ssbbartgroup.com/blog/csscontentproperty/). This is a good thing as the icons are used purely for stylistic purposes and should not be adding additional clutter to the things screen readers must read out. Maybe @Neurrone can weigh in too :)

**Comment by Neurrone** [comment]
@whipermr5 thanks for the mention and thinking about how it'd read. 

TLDR: good technical solution, not worth additional complexity for alternatives.

Long version:

That article is out-of-date; in [every browser except IE,](http://tink.uk/accessibility-support-for-css-generated-content/) pseudo-elements are read. The only way I know of hiding it is using the aria-hidden attribute, but it needs the icon to be in its own DOM node, so this approach can't be used in that case. Since its such a minor thing if its read, its probably not worth the extra complexity of creating a new node etc.

**Comment by Mynk96** [comment]
@damithc is there any change to be made in this PR?

**Comment by damithc** [comment]
@Neurrone are you recommending the css solution or the current solution?

> TLDR: this is a good technical solution, not worth additional complexity for alternatives.

**Comment by Neurrone** [comment]
@damithc sorry I meant that the CSS solution is the best one.

**Comment by damithc** [comment]
@Mynk96 do you plan to update the PR to use the CSS approach?

**Comment by Mynk96** [comment]
@damithc yes,I think that would be better approach.Should the commit the changes?

**Comment by damithc** [comment]
> Should the commit the changes?

Yes, go ahead.

**Comment by Mynk96** [comment]
@damithc My bad,Fine I'll change.

**Comment by Mynk96** [comment]
@damithc ready for review!

File: src/main/webapp/WEB-INF/tags/statusMessage.tag [hunk 1]
```diff
@@ -11,7 +11,7 @@
         <div id="statusMessagesToUser">
             <c:forEach items="${statusMessagesToUser}" var="statusMessage">
                 <div class="overflow-auto alert alert-${statusMessage.color} statusMessage">
-                    ${statusMessage.text}
+                        ${statusMessage.text}
```
Comment (whipermr5) [inline src/main/webapp/WEB-INF/tags/statusMessage.tag]: Unrelated whitespace change. Once this is removed I think the entire PR should collapse to a single changed file (`teammatesCommon.css`).

File: src/main/webapp/stylesheets/teammatesCommon.css [hunk 2]
```diff
@@ -1120,3 +1120,14 @@ Prevent Long URL From Breaking Out of Container
     float: left;
     margin-right: 5px;
 }
+
+/*
+Glyphicon for success status messages
+*/
+
```
Comment (whipermr5) [inline src/main/webapp/stylesheets/teammatesCommon.css]: Remove this blank line

**Comment by Mynk96** [comment]
@whipermr5 ready for review

**Comment by damithc** [comment]
@whipermr5 may be we can do the same for other type of status messages? Can be done as a separate PR.

File: src/main/webapp/stylesheets/teammatesCommon.css [hunk 3]
```diff
@@ -1150,3 +1150,13 @@ Prevent Long URL From Breaking Out of Container
     float: left;
     margin-right: 5px;
 }
+
+/*
+Glyphicon for success status messages
```
Comment (damithc) [inline src/main/webapp/stylesheets/teammatesCommon.css]: Did we miss a `*` here?

Comment (whipermr5) [inline src/main/webapp/stylesheets/teammatesCommon.css]: I thought so too, until I went to take a look at the rest of the file.
